### DISCUSSION TITLE
Dynamic key is expired

### DISCUSSION THREAD

**Discussion by youssef-mahdi** [netless-io/flat#1340] [General]
### I'm using agora web. Suddenly i faced an error which is dynamic key is expired code: 6 
does anyone had same issue?

**Comment by BlackHole1** [comment]
Hi @youssef-mahdi 👋. This is the error reported by agora-web-sdk-ng and is probably memory related.

Can you try refreshing the page? And see if it still exists after the refresh is complete. 🙇‍♂️
